en/08_Changelogs/6.0.0.md

```diff
@@ -916,12 +917,21 @@ SilverStripe\ORM\Connect\MySQLDatabase:
 
 ### `DBDecimal` default value {#dbdecimal-default-value}
 
-Previously if an invalid default value was provided for a [`DBDecimal`](api:SilverStripe\ORM\FieldType\DBDecimal) database column, it would silently set the defalt value to `0`. This will now throw an exception instead, so that you're aware your configured value is invalid and can correct it.
+Previously if an invalid default value was provided for a [`DBDecimal`](api:SilverStripe\ORM\FieldType\DBDecimal) database column, it would silently set the default value to `0`. This will now throw an exception instead, so that you're aware your configured value is invalid and can correct it.
 
 ### `RedirectorPage` validation
 
 [`RedirectorPage`](api:SilverStripe\CMS\Model\RedirectorPage) now uses the [`Url` constraint](https://symfony.com/doc/current/reference/constraints/Url.html) from `symfony/validator` to validate the `ExternalURL` field. It will no longer add `http://` to the start of URLs for you if you're missing a protocol - instead, a validation error message will be displayed.
 
+### Change JS MIME type & remove default `type` attribute in `<script>` tags {#change-js-mime-type}
```

Comment:
This is a pretty long heading, but I couldn't think of anything more succinct to suggest so it's probably okay.

Reply:
Changed it form:
Change JS MIME type & remove default `type` attribute in `<script>` tags {#change-js-mime-type}
to
### Update JS MIME type, remove `type` in `<script>` tags {#js-mime-type-update}